pkg/gotohelm/rewrite.go

```diff
@@ -67,26 +67,30 @@ func LoadPackages(cfg *packages.Config, patterns ...string) ([]*packages.Package
 			return nil, errors.Wrapf(errors.Join(errs...), "package %s", pkg.Name)
 		}
 
-		for _, parsed := range pkg.Syntax {
-			filename := pkg.Fset.File(parsed.Pos()).Name()
+		for _, fd := range pkgWithFilesTuple(pkg, deps) {
```

Comment:
I don't understand the need for this modification. Does calling `LoadPackages(cfg, append(mainpkg, deps...)...)` not work here?

Reply:
<img width="742" alt="Screenshot 2024-10-03 at 12 34 58" src="https://github.com/user-attachments/assets/e34a7f0b-6a93-44f6-81ef-fe1ed6e3966e">
When you provide multiple patterns to package.Load call, then as a result you have list of packages. 

That can work, but the shim "package" would be duplicated. Maybe this is better to exclude "sneaky" shim append call and make it a proper pkg translation call.